doc/setup_instructions.md

```diff
@@ -486,3 +485,117 @@ build tool option, if it isn't already.
 
 Your locally cloned Thrive version may get messed up from time to time.
 [Here are the steps to fix it.](https://wiki.revolutionarygamesstudio.com/wiki/Cleaning_Local_Thrive_Version)
+
+Working with translation
+========================
+
+Thrive is using gettext as a localization tool. gettext works by having a
+template file (of extension type .pot) and then one translation file for
+each language (of extension type .po)
+
+Required tools
+--------------
+
+If you are planning to do anything that would require translation, or
+simply to translate the game into your locale language you may need
+a few more tools.
+
+Note if you are simply planning to add a new localization, Poedit is
+enough.
+
+### Python
+
+The tool used to extract strings from the game files is using
+[Python](https://www.python.org/downloads).
+So if you are planning to add or edit strings into the game, you'll need it.
+
+### Babel and Babel_thrive
+
+Babel and its extension [Babel_thrive](https://github.com/westtunger/pybabel-godot)
+are the tools used for extracting strings from the game files.
+Just like Python, you'll want to download these if you are planning
+to add or edit strings into the game.
```

Comment:
Instructions? Example pip command?

Reply:
Those are in the readme on Github, that's why I didn't add these here too.
If you think I should, I'll do it.

Reply:
It'd be much quicker to read about it here, after all you just need to run a simple pip command, right?

Reply:
Indeed, I'll do that